cmd/minikube/cmd/config/unset_test.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import "testing"
+
+func TestUnsetConfig(t *testing.T) {
+	err := Set("dashboard", "true")
+	err = unset("dashboard")
+	if err != nil {
+		t.Fatalf("Failed to unset property ")
```

Comment:
Prefer t.Errorf over t.Fatalf whenever possible. That way you can see more failed test information without rerunning it later. In this particular case, it doesn't change the behavior.

Reply:
Thanks for pointing it out. I have updated it.